Problems & benefits

TryTooFly opened the pull request:
сітка елементів це я розумію дісплей грід? трохи з костилями через цей грід получається але як є + лічильник присутній. також нова тема для мене

TryTooFly committed:
початок роботи з problems

TryTooFly committed:
Update _problems.scss

TryTooFly committed:
problems and benefits mobile

TryTooFly committed:
work

kognitiive commented on src/partials/benefits.html:
```diff
@@ -0,0 +1,18 @@
+<section class="benefits">
+  <div class="container">
+    <h2 class="benefits__title">
+      Why do people<br />
```
Тут br не потрібен я думаю, спробуй через width

kognitiive commented on src/partials/benefits.html:
```diff
@@ -0,0 +1,18 @@
+      choose us
+    </h2>
+    <div class="benefits__list">
+      <p class="benefits-list__item"><br />Training in small groups</p>
```
Теж br, якщо вони треба напиши навіщо, будемо пробувати без них)

kognitiive commented on src/partials/problems.html:
```diff
@@ -0,0 +1,17 @@
+<section class="problems">
+  <div class="container">
+    <h2 class="problems__title">
+      The main problems of students<br />
```
Тут також з br будемо розбиратись

kognitiive commented on src/sass/_benefits.scss:
```diff
@@ -0,0 +1,45 @@
+.benefits__title {
+  font-weight: 800;
+  font-size: 24px;
+  line-height: 130%;
```
line-height через множник

kognitiive commented on src/sass/_benefits.scss:
```diff
@@ -0,0 +1,45 @@
+  color: var(--black-color);
+}
+.benefits-list__item::before {
+  display: block;
```
display: inline-block;

kognitiive commented on src/sass/_benefits.scss:
```diff
@@ -0,0 +1,45 @@
+  line-height: 1.2;
+  letter-spacing: 0.08em;
+  color: var(--accent-color);
+  margin-bottom: -13px;
```
Тут прибрати margin поки що

kognitiive commented on src/sass/_problems.scss:
```diff
@@ -0,0 +1,45 @@
+.problems__title {
+  font-weight: 800;
+  font-size: 24px;
+  line-height: 130%;
```
line-height через множник

kognitiive reviewed (commented):
Подивись властивість gap, якщо там довго то на display:flex, але починаючи з 
min-width:768px

TryTooFly committed:
work

TryTooFly committed:
робота

TryTooFly committed:
done

TryTooFly committed:
готово

забрав беери. Проблемс і бенефітс зробив одинакову розміку на компютерній версії, так гарніше буде

kognitiive committed:
Merge branch 'main' into problems-&-benefits
